### ISSUE TITLE
Flip magenta and yellow? Change colors back to red/green?

### ISSUE THREAD

**Issue by mcmire** [splitwise/super_diff#39]
When printing:

```
Expected: ...
       to eq: ...
```

The "actual" value is in red and the "expected" value is in green. Shouldn't it be the other way around? (i.e. the actual value should be assumed to be the right one).

Note that this means we'd have to flip the colors in the diff as well, and update the legend.

**Comment by mcmire** [comment]
Not sure why I got confused here. It already works like I thought it did.

**Comment by mcmire** [comment]
Reopening this. It feels like more often than not, the actual value is more important than the expected value, so it should jump out more when looking at the output. This will cause the diff to look weird, though, so maybe not?

**Comment by mcmire** [comment]
This is what Jest does, for reference:

<img width="893" alt="Screen Shot 2019-12-09 at 9 33 11 PM" src="https://user-images.githubusercontent.com/7371/70495701-993fcd00-1acb-11ea-8603-a1a7537db9b4.png">

**Comment by mcmire** [comment]
Maybe worth looking at Ava too: https://github.com/avajs/ava#magic-assert

**Comment by hedgesky** [comment]
Would it make sense to make this configurable? Thus, the cost of the decision for the question in the title wouldn't be that high, because end users would be able to adjust color scheme to their preference if they'd like to.

For example, something like this:
```ruby
SuperDiff.configure do |config|
  config.set_color_scheme(SuperDiff::ColorSchemes::RedGreen) 
  # or MagentaYellow, or YellowMagenta
end
```

With schemes defined either similar to as they do now:
```ruby
module MagentaYellow
  COLORS = {
    alpha: :magenta,
    beta: :yellow,
    border: :blue,
    header: :white,
  }.freeze
end
```

or without relying on hashes:
```ruby
module MagentaYellow
  def self.alpha
    :magenta
  end

  def self.beta
    :yellow
  end

  def self.border
    :blue
  end

  def self.header
    :white
  end
end
```

As another option, `set_color_scheme` could accept symbol instead of a module as an argument. That approach would be less verbose, but less extensible.

**Comment by mcmire** [comment]
Yeah, that's a good point, perhaps the user is the best judge here. I like the idea of passing a module. Or perhaps just a hash?

``` ruby
SuperDiff.configure do |config|
  config.color_scheme = SuperDiff::ColorSchemes.red_green
  # or magenta_yellow, or yellow_magenta... or supply your own
end
```

or if you wanted to change a couple of settings on the fly, you could even do:

``` ruby
SuperDiff.configure do |config|
  # color_scheme is set to red_green by default
  config.color_scheme.merge!(alpha: :magenta, beta: :yellow)
end
```

**Comment by hedgesky** [comment]
Just to note, I suggested `.set_color_scheme` method name as it (for my taste, at least) is a bit more aligned with `. add_extra_differ_class` and other already used configuration methods. 

Regarding passing hashes and encouraging users to merge values into them: it's nice and simple solution. A downside: users could rely on the fact that it's a hash, making major changes to color scheme functionalities more difficult.
